completions/ssh

```diff
@@ -55,7 +55,7 @@ _comp_xfunc_ssh_options()
         NoHostAuthenticationForLocalhost NumberOfPasswordPrompts
         PasswordAuthentication PermitLocalCommand PKCS11Provider Port
         PreferredAuthentications ProxyCommand ProxyJump ProxyUseFdpass
-        PubkeyAcceptedKeyTypes PubkeyAuthentication RekeyLimit RemoteCommand
+        PubkeyAcceptedAlgorithms PubkeyAuthentication RekeyLimit RemoteCommand
```

Comment:
The change in question seems to be a fairly recent one, so I think it would be better to add `PubkeyAcceptedKeyTypes` to the "Selected old ones" list below this one, instead of removing it completely.